packages/server/src/jsRunner.ts

```diff
@@ -1,61 +1,145 @@
-import vm from "vm"
+import ivm from "isolated-vm"
```

Comment:
When me you and @mike12345567 chatted about this, I remember Mike having the idea to have a test suite in `package/server` that calls this `jsRunner.ts` code with JS snippets containing calls to every helper. Is that something we could do now we have easy access to the JS helper list from `string-templates`?

Reply:
Done in here https://github.com/Budibase/budibase/pull/12922